indigo-frontend/src/app/pages/notebook/notebook-info/notebook-info.component.html

```diff
@@ -0,0 +1,43 @@
+@if (isLoading) {
+<div class="flex items-center justify-center py-8">
+    <span>Loading notebook...</span>
+</div>
+} @else if (hasError) {
+<div class="flex items-center justify-center py-8">
+    <div class="text-center">
+        <span>There was an error loading the notebook, please try again later</span>
+    </div>
+</div>
+} @else if (notebook) {
+<div class="grid grid-cols-1 gap-4 pt-4">
+    <eln-card>
+        <div class="px-3">
+            <div class="flex items-center justify-between pb-3 border-b border-b-neutral-200">
+                <h2 class="font-semibold text-base">About Notebook</h2>
+                <eln-button variant="blue-outline" classList="text-primary-400" [disabled]="true">
+                    <em class="indicon-edit"></em>
+                    Edit
+                </eln-button>
+            </div>
+
+            <div class="space-y-6 mt-3 text-sm">
+                <div>
+                    <h3 class="font-semibold text-neutral-800 mb-2">Notebook Name</h3>
+                    <input type="text"
```

Comment:
Seems that we don't need input here and textarea below as this is just text fields, so we can proceed with simple divs. Text inputs will be in edit dialog.